tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -236,11 +239,6 @@ class GivenUsingSuite extends BaseDottySuite {
     )
   }
 
-  test("given-subtype-error".ignore) {
-    // it is treaten as alias without '=' sign at the end and {...} is refinement part
-    runTestError("given intOrd: ? <: Ord[Int] { def f(): Int = 1 }", "missing = at the end")
```

Comment:
This doesn't parse in Scala 3, so I removed the test case. We do parse it in both Scala 2 and 3, but I think form the parser perspective it is fine. It's also the check needed here would be a bit complicated.